frontend/layouts/discover-page/navigation/component.tsx

```diff
@@ -0,0 +1,56 @@
+import { FC } from 'react';
+
+import { useIntl } from 'react-intl';
+
+import { useRouter } from 'next/router';
+
+import BadgeNavigation from 'components/badge-navigation';
+import { Paths } from 'enums';
+
+import { NavigationProps } from './types';
+
+export const Navigation: FC<NavigationProps> = ({ stats }: NavigationProps) => {
+  const intl = useIntl();
+  const { asPath } = useRouter();
+
+  const navigationItems = [
```

Comment:
If you have performed a search and change tab, the search is not maintained. We don't need to fix this now but we'll need it later (same will apply to the filters).